docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -96,13 +96,19 @@ Set up Aiven services
 Set up sample data
 ------------------
 
+These steps show you how to create sample records to provide streamed data that is processed by the data pipelines presented in this tutorial. You can also use other existing data, although many of the examples in this tutorial are based on the use of this sample data.
+
 Before you start, clone the `Dockerized fake data producer for Aiven for Apache Kafka <https://github.com/aiven/fake-data-producer-for-apache-kafka-docker>`_ Git repository to your computer.
 
 1. Follow `these instructions <https://developer.aiven.io/docs/tools/cli/user/user-access-token.html#manage-access-tokens>`_ to create an authentication token for your Aiven account.
 
+   This is required to allow the tool to connect to a service in your Aiven account.
+
 #. Go to the data producer tool directory and copy the ``conf/env.conf.sample`` file to ``conf/env.conf``.
 
-#. Edit the ``conf/env.conf`` file and update the parameters:
+#. Edit the ``conf/env.conf`` file and update the parameters.
+
+   Replace ``AIVEN_PROJECT_NAME`` and ``AIVEN_ACCOUNT_EMAIL`` with the details for your Aiven account, and replace ``AUTHENTICATION_TOKEN`` with the token that you created.
```

Comment:
Should we add a mention to check the github readme file for more info? (Also if something changes in the repo itself in the future)